DependencyInjection/Configuration.php

```diff
@@ -32,14 +32,12 @@ public function getConfigTreeBuilder()
             ->children()
                 ->arrayNode('directories')
                     ->info('List of directories relative to the kernel root directory containing classes.')
-                    ->useAttributeAsKey('prefix')
-                    ->prototype('array')
-                        ->prototype('scalar')->end()
-                    ->end()
+                    ->prototype('scalar')->end()
                     ->defaultValue([
-                        'controller' => ['../src/*Bundle/Controller', '../src/*Bundle/Action'],
-                        'command' => ['../src/*Bundle/Command'],
-                        'event_subscriber' => ['../src/*Bundle/EventSubscriber'],
+                        '../src/*Bundle/Controller',
+                        '../src/*Bundle/Action',
+                        '../src/*Bundle/Command',
+                        '../src/*Bundle/EventSubscriber',
```

Comment:
I agree with tout.

Maybe can we enable by default only `Controller` (it's already a magical directory) and `Action` (it's a new one, basically the same as `Controller/`) and Command (it's already a magic directory) and only explain in the doc how to enable event listeners and others.

WDYT @Ener-Getick?